.azure-pipelines/scripts/codeScan/pyspelling/inc_dict.txt

```diff
@@ -1331,6 +1331,8 @@ OTHpart
 Ott
 oup
 outdir
+Outlier
```

Comment:
why this word is putting into the skip word list? Apparently the word is wrong, you should use "The outlier" or "An outlier".